### DISCUSSION TITLE
Positivity (to the devs)

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#8818] [appreciation]
So, I know you guys are probably getting a lot of crap lately. I just want to let you know that I still love this mod, and am grateful for the quick fix with the last update, and all the work being put into this.

Thank you, and remember to always try and stay positive.  =)

**Comment by MaximumRustler** [comment]
I feel the same way. 😃

**Comment by AgentZephyr** [comment]
Ditto.  Keep up the awesome work you guys!
